lib/ach/records/batch_control.rb

```diff
@@ -13,10 +13,8 @@ class BatchControl < Record
     field :entry_hash, Integer, lambda { |f| sprintf('%010d', f % (10 ** 10))}
     field :debit_total, Integer, lambda { |f| sprintf('%012d', f)}
     field :credit_total, Integer, lambda { |f| sprintf('%012d', f)}
-    field :company_identification_code_designator, String, nil, '1',
-        /\A[0-9A-Z ]\z/
     field :company_identification, String,
-        lambda { |f| f.rjust(9) }, nil, /\A[\dA-Z]\d{6,8}\z/
+        lambda { |f| f.length == 10 ? f : "1#{f}" }, nil, /\A[\da-zA-Z ]{9,10}\z/
```

Comment:
This is different from upstream in that it allows letters and spaces throughout the string. Not sure why, because this alignes with what's allowed below in `batch_header.rb` for this field (and *is* this way upstream).